src/api/messages/ItemPriceMessage.ts

```diff
@@ -0,0 +1,14 @@
+import { IsNotEmpty } from 'class-validator';
```

Comment:
I dont think we have ItemPriceMessages that we receive from the network, I have no idea what this is or what it is used for.